cmd/newtemplates/browser.js

```diff
@@ -0,0 +1,60 @@
+import { browser } from "k6/browser";
+import http from "k6/http";
+import { sleep, check } from 'k6';
+
+const BASE_URL = __ENV.BASE_URL || "https://quickpizza.grafana.com";
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: "shared-iterations",
+      options: {
+        browser: {
+          type: "chromium",
+        },
+      },
+    },
+  },{{ if .ProjectID }}
+  cloud: {
+    projectID: {{ .ProjectID }},
+    name: "{{ .ScriptName }}",
+  },{{ end }}
+};
+
+export function setup() {
+  let res = http.get(BASE_URL);
+  if (res.status !== 200) {
+    throw new Error(
+      `Got unexpected status code ${res.status} when trying to setup. Exiting.`
+    );
+  }
+}
+
+export default async function() {
+  let checkData;
+  const page = await browser.newPage();
+
+  try {
```

Comment:
Would adding a `catch` here a recommended practice? 🤔 
If so, I'd prefer to have it in place, as this not being a simple test, but likely an entrypoint for k6 newcomers.

Reply:
I believe in this case, if the error is not caught, the iteration will be aborted with an error anyway. But it would make sense indeed, if possible to catch the error in order to maybe explicitly call `fail` or `exec.test.abort` with a custom message guiding the user through what failed, why, and what they can do about it 👍🏻

Reply:
I have added catch + fail :eyes: 

Also, another thing we can maybe use in all examples cc @ankur22